packages/gatsby-source-shopify/src/__tests__/index.js

```diff
@@ -15,12 +16,7 @@ jest.mock(`../create-client`, () => {
             const [queryName] = Object.entries(mockQueries).find(
               ([queryName, queryString]) => queryString === query
             )
```

Comment:
> Hack alert.

if the query is a generated string and not the string exact from the `defaultQueries`, then this gets false: `queryString === new String(query)`

i don't know if this is more or less hacky:
```suggestion
            const queryName = /query Get(\w*)/.exec(query)[1].toLowerCase()
```
*works with rename `shoppolicies.json` -> `policies.json`*